### ISSUE TITLE
Train with custom dataset error

### ISSUE THREAD

**Issue by localryu** [haotian-liu/yolact_edge#162]
**Describe the bug**
I got 'Warning: Moving average ignored a value of inf/non' error when I train with my custom data.
I think the difference image size(form coco data) brings this error. Is it right? Then, How can i train with my custom data... 

**To Reproduce**
1. For using my custom data, I changed config file as below
- (data-set image size is 1920x1080)

```
ryu_dataset = dataset_base.copy({
    'name': 'RYU Dataset',
    
    'train_images': './data/ryu/images',
    'train_info':   './data/ryu/annotations/ryu_trainval.json',

    'valid_images': './data/ryu/images',
    'valid_info':   './data/ryu/annotations/ryu_trainval.json',

    'has_gt': True,

    'class_names': ('road',),
    'label_map': {1:1,}
})
```

```
yolact_edge_mobilenetv2_config_for_ryu_config = yolact_edge_config.copy({
    'name': 'yolact_edge_mobilenetv2_config_for_ryu_config',

    # Dataset stuff
    'dataset': ryu_dataset,
    'num_classes': len(ryu_dataset.class_names) + 1,

    'lr_steps': (2800, 6000, 7000, 7500),
    'max_iter': 8000,
    'backbone': mobilenetv2_backbone
})
```
the image size is different from coco dataset, i changed some parameters in config file.
`'max_size': 1920,` 
and
```
    'discard_box_width': 4 / 1920,
    'discard_box_height': 4 / 1920,
```
2. Train with pre-trained model
`python3 train.py --batch_size=2 --num_workers=0  --config=yolact_edge_mobilenetv2_config_for_ryu_config --dataset=ryu_dataset --resume=weights/mobilenet_v2-b0353104.pth --log=./logs`

**Full logs**
[11/03 13:53:19 yolact.train]: eta: 1:39:18  epoch: 0  iter: 70  B: 25498131216940012028745295442477056.000  M: 30.639  C: 141917513198487.281  S: 98355576.799  T: 25498131216940012028745295442477056.000  time: 0.628  data_time: 0.247  lr: 0.001000  max_mem: 6011M
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of nan

**Environment:**
 - OS: Ubuntu 18.04
 - GPU:  RTX 2080
 - CUDA Version 10.1

**Comment by haotian-liu** [comment]
For custom dataset, you might need to tune the learning rate.  You can also use a longer lr warmup.

**Comment by localryu** [comment]
@haotian-liu thank you.  It works for me.

**Comment by PrajwalCogniac** [comment]
Hey @localryu I am facing the exact same issue. What did you do to fix the issue ?

**Comment by PrajwalCogniac** [comment]
@haotian-liu is there any response to this ?
